### PR TITLE
Update to SDK 0.34.0

### PR DESCRIPTION
**Pull request by rflechtner**
Updates the cType fetching example to the latest SDK version.

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [x] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

File: code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts [hunk 1]
```diff
@@ -2,12 +2,11 @@ import * as Kilt from '@kiltprotocol/sdk-js'
 
 /* eslint-disable @typescript-eslint/no-unused-vars */
 export async function main(): Promise<Kilt.ICType> {
-  const { creator, createdAt, ...domainLinkageCType } =
-    await Kilt.CType.fetchFromChain(
-      'kilt:ctype:0xb08800a574c436831a2b9fce00fd16e9df489b2b3695e88a0895d148eca0311e'
-    )
+  const { creator, createdAt, cType } = await Kilt.CType.fetchFromChain(
```
Comment (Dudleyneedham) [inline code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts]: shouldn't we keep the name the same as it is an example?

```suggestion
  const { creator, createdAt, cType: domainLinkageCType } = await Kilt.CType.fetchFromChain(
```

File: code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts [hunk 2]
```diff
@@ -2,12 +2,11 @@ import * as Kilt from '@kiltprotocol/sdk-js'
...
+    'kilt:ctype:0xb08800a574c436831a2b9fce00fd16e9df489b2b3695e88a0895d148eca0311e'
+  )
 
-  console.log(JSON.stringify(domainLinkageCType, null, 2))
+  console.log(JSON.stringify(cType, null, 2))
```
Comment (Dudleyneedham) [inline code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts]:
```suggestion
  console.log(JSON.stringify(domainLinkageCType, null, 2))
```

File: code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts [hunk 3]
```diff
@@ -26,5 +25,5 @@ export async function main(): Promise<Kilt.ICType> {
     "$id": "kilt:ctype:0xb08800a574c436831a2b9fce00fd16e9df489b2b3695e88a0895d148eca0311e"
   }
   */
-  return domainLinkageCType
+  return cType
```
Comment (Dudleyneedham) [inline code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts]:
```suggestion
  return domainLinkageCType
```

**Review by Dudleyneedham — CHANGES_REQUESTED** [review]
Minor changes, but I think you haven't checked out the workshop.

Please can you do this as well.

File: code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts (hunk 1 above)
Comment (rflechtner) [reply code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts]: why tho????

Comment (rflechtner) [reply code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts]: nobody should be using this CType btw, it is wrong and broken

Comment (rflechtner) [reply code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts]: definitely not what people should use for the domain linkage credential

**Comment by rflechtner** [comment]
> Minor changes, but I think you haven't checked out the workshop.
> 
> Please can you do this as well.

There is no CType fetching in the workshop

File: code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts (hunk 1 above)
Comment (rflechtner) [reply code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts]: ok reverted now, guess the issue that this is a shitty ctype is a separate issue which has also wider implications, as it seems to be the one everybody uses.

**Review by Ad96el — APPROVED** [review]

**Review by Dudleyneedham — APPROVED** [review]
MUCH WOW

**Pull request merged**